lib/entropy/fetchers.py

```diff
@@ -596,8 +596,9 @@ def _urllib_download(self):
 
             # Handle HTTP Basic auth
             if self.__http_basic_user and self.__http_basic_pwd:
-                basic_header = base64.encodestring('%s:%s' % (
-                    self.__http_basic_user, self.__http_basic_pwd)).replace('\n', '')
+                basic_header = base64.b64encode(('%s:%s' %
```

Comment:
In Entropy it is preferred to do it more like: `xx = '%s' % ( # after this goes a newline`.